client/src/components/ReadStatus.js

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+
+export const DropDown = styled.div`
+  border-width: 1px;
+  padding: 6px 0 7px 8px;
+  width: 105px;
+  font-size: 13px;
+  font-weight: bold;
+  font-family: "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  background: #f2f2f2;
+  border-color: #dddddd;
+  text-align: left;
+  white-space: nowrap;
+  color: #181818;
+  text-overflow: ellipsis;
+  border-bottom-left-radius: 3px;
+  border-top-left-radius: 3px;
+  border-style: solid;
+  display: flex;
+  align-items: center;
+`
+
+export const RightButton = styled.div`
+  background-color: #409D69;
+  color: #fff;
+  border-width: 0px;
+  width: 27px;
+  font-size: 13px;
+  border-bottom-right-radius: 3px;
+  border-top-right-radius: 3px;
+  border-left: 1px solid #38883d;
+  margin-right: -10px;
+  padding: 0;
+`
+
+export const ShelfButton = styled.button`
+  box-sizing: border-box;
+  background-color: transparent;
+  background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAICAQAAABaf7ccAAAAKUlEQ…3KsQEAIAzDsJzez8MuoF6tdLo1icQdiBvilrglnw1xS9wHeq2Hge3+H0sAAAAASUVORK5CYII=);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: 8px 4px;
+  border: 0;
+  cursor: pointer;
+  display: block;
+  font-family: "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
I think you could put some CSS stylings into higher order component to prevent having to repeatedly write the same stylings for children components. For example, you may want to consider putting font-family a higher level component so it applies to all children.